### PR TITLE
Added 4 apps & fixed typos

### PR DESCRIPTION
**Pull request by Maryom**
Sorry I don't know why it failed :( I tried a lot and it didn't pass. Inshaallah I will give it another try ASAP maybe tomorrow.

### CODE REVIEW

**Comment by Mazyod** [comment]
Thanks! Please also review the comments on the changes

File: README.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
 
 <h1 align="center">
-  Appz :iphone:
-  <h6 align="center">
-    Deeplinking to external applications made easy
-  </h6>
```
Comment (Mazyod) [inline README.md]: Please keep the indentation here as it was

File: README.md [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
-  <img alt="Version" src="https://img.shields.io/badge/version-2.0.1-blue.svg"/>
-  <a alt="Travis CI" href="https://travis-ci.org/SwiftKitz/Appz">
-    <img alt="Version" src="https://travis-ci.org/SwiftKitz/Appz.svg?branch=master"/>
-  </a>
-  <img alt="Swift" src="https://img.shields.io/badge/swift-3.0-orange.svg"/>
-  <img alt="Platforms" src="https://img.shields.io/badge/platform-ios%20%7C%20watchos%20%7C%20tvos-lightgrey.svg"/>
-  <a alt="Carthage Compatible" href="https://github.com/SwiftKitz/Appz#carthage">
-    <img alt="Carthage" src="https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat"/>
-  </a>
-  <img alt="Supported Apps" src="https://img.shields.io/badge/Apps-156-9600cd.svg"/>
```
Comment (Mazyod) [inline README.md]: Also here, please keep it as it was

File: README.md [hunk 3]
```diff
@@ -65,34 +65,34 @@ __Add your applications:__
...
+// Each action should provide an app path and web path to be
+// added to the associated URL
+var paths: ActionPaths {
 
-            switch self {
-            case .open:
-                return ActionPaths()
-            }
-        }
-    }
```
Comment (Mazyod) [inline README.md]: All these indentations should be kept as it was

File: README.md [hunk 4]
```diff
@@ -458,6 +465,7 @@ Appz is released under the MIT license. See LICENSE for details.
 [Vox-link]: http://coppertino.com/vox/iphone
 [Voxer-link]: http://www.voxer.com
 [VSCO-link]: http://vsco.co/store/app
+[Watch-link]: https://www.reddit.com/r/workflow/comments/3mux7h/ios_url_schemes/
```
Comment (Mazyod) [inline README.md]: Please rename it to AppleWatch

**Comment by Mazyod** [comment]
Awesome! Also Travis is finally happy :D

**Comment by Maryom** [comment]
@Mazyod At your service. I have a question if you don't mind: I noticed in `extension NSExtensionContext: ApplicationCaller` in line 22 this comment:

`// maybe use a semaphore instead`

I think it's interesting I searched about it and I have an overview about it. But I didn't get how we can use it in Appz 📱 any clue ?

**Comment by Mazyod** [comment]
Nevermind that, a semaphore probably would cause a deadlock since the completion handler might be scheduled to run on the main thread anyway. I don't think anyone is using it anyway.

**Pull request merged**